lib/web/ui/server.go

```diff
@@ -193,3 +193,39 @@ func MakeDatabases(clusterName string, databases []types.Database) []Database {
 
 	return uiServers
 }
+
+// TODO: move this stuff to a more sensible constants file
+// OS is the string representation of the major operating systems we support.
+type OS string
+
+const (
+	Windows OS = "windows"
+)
+
+// Desktop describes a desktop to pass to the ui.
+type Desktop struct {
+	// OS is the os of this desktop
+	OS OS `json:"os"`
+	// Name is name (uuid) of the windows desktop.
```

Comment:
```suggestion
	// Name is name (uuid) of the desktop.
``` 

Not necessarily Windows, right?